### PR TITLE
NMS-13057: Add PostgreSQL scram authentication hint

### PR DESCRIPTION
**Pull request by mfuhrmann**
### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13057

### CODE REVIEW

File: docs/modules/deployment/pages/core/getting-started.adoc [hunk 1]
```diff
@@ -37,6 +37,8 @@ include::../time-sync.adoc[]
 [[setup-postgresql]]
 == Set up PostgreSQL
 
+TIP: PostgreSQL 14 uses https://www.postgresql.org/docs/14/auth-password.html[password authentication] method `scram-sha-256` by default. If you use older versions of PostgreSQL you should change the method in *postgresql.conf* and in *pg_hba.conf* before continuing with <<install-core-instance,Install the core instance>>.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/core/getting-started.adoc:40]:
```suggestion
TIP: PostgreSQL 14 uses https://www.postgresql.org/docs/14/auth-password.html[password authentication] method `scram-sha-256` by default.
If you use older versions of PostgreSQL, you should change the method in *postgresql.conf* and in *pg_hba.conf* before continuing with <<install-core-instance,Install the core instance>>.
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]
One minor change. I was also wondering - should this PR be targeting `foundation-2022` instead of `develop`?

**Comment by Bonrob2** [comment]
> One minor change. I was also wondering - should this PR be targeting `foundation-2022` instead of `develop`?

Yes! Good catch. Please target Foundation-2022.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Removed bold from the file names.

**Review by mmahacek — DISMISSED** [review]

**Comment by emarsh1995** [comment]
Updated the base to foundation-2022, so I'm gonna check that nothing broke.

**Comment by emarsh1995** [comment]
Changed it back to develop because foundation-2022 wanted to add some 2,000 commits. 😓

**Comment by Bonrob2** [comment]
Closing PR due to rebase issue. New PR is here: https://github.com/OpenNMS/opennms/pull/5163

**Comment by mfuhrmann** [comment]
Thanks for working on this!

**Comment by Bonrob2** [comment]
> Thanks for working on this!

Thanks for getting most of it done before you left!

**Pull request closed without merging**